### DISCUSSION TITLE
External player

### DISCUSSION THREAD

**Discussion by ghost** [community/community#119700] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### External device playback

Hi,
I have managed to turn **off** "allow external devices to start playback" on my Samsung Music App as I did not want playback to auto start when I connected my phone to the car for sat nav. 
I'm wondering if this option will also prevent auto playback from the Voice App as I cannot find anything similar on the app. 
Until now, auto playback starts from whichever media I used last, be it Samsung Music or Voice. I want neither to do this. 
Any help appreciated. Thanks.
![SmartSelect_20240415_164742_Samsung Music](https://github.com/community/community/assets/167112916/2a2d3cac-9371-4145-ba10-d6d1060f1188)

**Comment by AlanGreene** [comment]
@Sult100 I'm not sure how issues with the Samsung Music app are related to GitHub. Did you create this discussion in the wrong place perhaps?

**Comment by ebndev** [comment]
Hi @Sult100 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking somewhere that focuses on your issue. GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

What that being said, I will be closing this discussion.

Best of luck!
